c2corg_ui/static/js/user.js

```diff
@@ -94,4 +94,24 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
```

Comment:
I don't think the 2 following functions are still needed (conflict merge error I think). @gberaudo has moved them in an ``apiService``, see the function above.